apps/stirling-pdf/0.33.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.33.1
+    image: frooodle/s-pdf:0.34.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There doesn't appear to be any major code discrepancies between the two versions provided. If you have concerns about potential issues or inefficiencies that may need addressing, kindly share more details!